### PR TITLE
device identity

### PR DESCRIPTION
**Pull request by piotrbartman**
implements QubesOS/qubes-issues/issues/9325

### CODE REVIEW

**Comment by marmarek** [comment]
Changes here looks okay, but I haven't tried running this version yet.

**Comment by piotrbartman** [comment]
Important note on breaking backward compatibility: there may exists scripts using `qvm-device devclass attach VM back:port --persistent` to assign the device (which still works) and respectively `qvm-device detach VM back:port` to remove this assignment, which does not work anymore!

**Comment by marmarek** [comment]
```
root@dom0:~# qvm-pci
Traceback (most recent call last):
  File "/usr/bin/qvm-pci", line 5, in <module>
    sys.exit(main())
             ~~~~^^
  File "/usr/lib/python3.13/site-packages/qubesadmin/tools/qvm_device.py", line 751, in main
    args.func(args)
    ~~~~~~~~~^^^^^^
  File "/usr/lib/python3.13/site-packages/qubesadmin/tools/qvm_device.py", line 101, in list_devices
    if args.assignments:
       ^^^^^^^^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'assignments'
```

**Comment by marmarek** [comment]
but `qvm-pci ls` works

File: qubesadmin/device_protocol.py [hunk 1]
```diff
@@ -284,10 +643,16 @@ class DeviceCategory(Enum):
...
+        "u01****",
+        "u0e****",
+        "u06****",
+        "u10****",
+        "p03****",
+        "p04****",
+    )
     Wireless = ("ue0****", "p0d****")
     Bluetooth = ("ue00101", "p0d11**")
     Mass_Data = ("b******", "u08****", "p01****")
```
Comment (marmarek) [inline qubesadmin/device_protocol.py]: I'm not sure which PR is the right one to comment, but "Mass_Data" prefix in devices widget is IMO not very intuitive (but should be fine on the API level). Maybe better "Storage" or "Disk"? @marmarta any other ideas? This is how it looks: https://openqa.qubes-os.org/tests/116595#step/qui_widgets_devices/6

Comment (fepitre) [reply qubesadmin/device_protocol.py]: Yes Mass_Data is not something useful, I would go with `Storage`.

Comment (piotrbartman) [reply qubesadmin/device_protocol.py]: I agree, it is already changed ;)

Comment (fepitre) [reply qubesadmin/device_protocol.py]: Sorry for the noise, I saw it after my comment.

**Comment by marmarek** [comment]
Categories cont. - see https://openqa.qubes-os.org/tests/118678#step/qubesmanager_vmsettings/60
The "Multimedia: unknown" is actually a graphics card, lspci shows "VGA compatible controller" (the unknown part is correct, it's a dummy ID)

**Review by marmarek — APPROVED** [review]

**Pull request merged**